content/community/governance/gcb/index.md

```diff
@@ -0,0 +1,102 @@
+# Galaxy Community Board Proposal
+
+**Keywords:** Special Interest Groups, Community of Practices
+
+## Origins
+
+The Galaxy Community Board (GCB) is proposed to take over the functions of the Global Galaxy Steering Committee.
+During GCC 2023, we discussed how to empower the Galaxy community in terms of project planning and effective execution, decision making and governance. 
+
+## Purposes
+
+Representation of community aspects of the ecosystem is central to long-term project sustainability. The goal is to empower community members to participate in planning and governance, through efficient and transparent structures and processes, that can scale effectively as the Galaxy project continues to expand. 
+
+The GCB will provide a supportive virtual forum for the exchange of ideas, and a governance body representing the interests of participating communities. It will organise and manage its own agenda and work, including arranging meetings and asynchronous discussions. The agenda might include planning new initiatives; supporting ongoing development; identifying and advocating for user needs; managing coherent, community-owned proposals and responses in the roadmap processes and other important project decision-making.
+
+Efficient communication between the GCB (Community Board) and both the GEB (Executive Board) and the GTB (Technical Board) requires at least one representative from GEB and from GTB to be also members of the GCB. It is recommended that at least one member of each of the other Boards are also members of the GCB, at all times.
+
+In summary, the goals of the GCB are:
+
+* discuss and summarize scientist (user) feedback to help guide the technical development of the Galaxy platform;
+* communicate scientist (user) vision and feedback to the GTB and GEB;
+* develop proposals to advance scientist (user) goals in the Galaxy community.
+
+## Community Special Interest Groups (SIGs)
+
+The GCB is a governance structure representing groups that are not primarily technical in nature, such as existing Proteomics and Microbial communities, though engaging in capacity building, toolkit development, outreach and training. The term *[Special Interest Group](/community/sig)* (SIG) is proposed as a term referring to these kinds of groups. No specific restrictions are proposed, other than making a useful contribution to the Project, so collaborating commercial users could form and manage their own SIG and gain access to the Roadmap process. 
+
+Active contributing communities are encouraged to form and govern their own SIG, with a focus on specific shared interests. Each SIG may elect one member to represent their interests on the GCB. This is analogous to the Technical Board, consisting of representatives of the Working Groups (WG).
```

Comment:
It's becoming tricky to distinguish WG from SIG, so would you want to stipulate something like 'consisting of representatives of the more centrally-funded Working Groups (WG)'?

Reply:
Keep the last sentence, that works and that makes sense, and if people want to learn more they can click on SIGs or WGs pages